src/main/java/org/springframework/social/partnercenter/api/PartnerCenterResponse.java

```diff
@@ -58,12 +61,23 @@ public PartnerCenterResponse setLinks(ResourceLinks links) {
 	}
 
 	public String getContinuationToken() {
-		return continuationToken;
+		return extractContinuationToken().orElse(null);
 	}
 
-	public PartnerCenterResponse setContinuationToken(String continuationToken) {
```

Comment:
Was this used somewhere?

Reply:
> Was this used somewhere?
setContinuationToken() no usages in monolith till now.